flow/__init__.py

```diff
@@ -25,6 +25,22 @@
 from .util.misc import redirect_log
 from .operations import with_job
 
+# Enable flow environments on an opt-out basis. Users must set the relevant
+# configuration variable to avoid environment being imported. This may be done
+# on either a global or a local configuration level. The order of precedence is
+# local up to global.
+from signac import get_project as _get_project
+
+# Work up in the correct order
+try:
+    flag = _get_project().config['flow'].get('IMPORT_ENVIRONMENTS')
```

Comment:
Sorry, I did not see that earlier. So far all configuration values have been snake_case. So I'd recommend to change the config key name to *auto_import_environments*.

Please rename `flag` to `auto_import_environments` and add `True` as the default value to the `get()` call.